.env

```diff
@@ -0,0 +1 @@
+JWT_SECRET_KEY="secret"
```

Comment:
So this .env file was committed just to show an example. Even the secret key string is an example. Because this won't be the key we will be using.  Just like you mentioned, I will generate a secret key, thought of 64 bit. 

> Will you setup the CI/CD in a separate PR right? Would suggest setting it up and getting done with it. You can use the same deployment structure as quiz backend.
Yes, I will.